packages/cypress/package.json

```diff
@@ -52,7 +52,7 @@
     "prebuild": "yarn clean",
     "build": "yarn prebuild && tsup",
     "build:watch": "yarn build --watch",
-    "test:cypress": "start-server-and-test 'yarn run test:server' 3000 'yarn run test:do-cypress'",
+    "test:cypress": "start-test 'yarn run dev:server' 3000 'yarn run dev:storybook' 6006 'yarn run test:do-cypress'",
```

Comment:
I know they are equivalent, but I prefer `start-server-and-test` to `start-test` because it's not clear what `start-test` is. Trying to figure out where it's coming from by searching for it will yield nothing. However, searching for `start-server-and-test` will at least find the dependency, which is a good hint as to where that is coming from.

Reply:
Yea, I agree with you. I wrote it that way because of the [docs](https://github.com/bahmutov/start-server-and-test?tab=readme-ov-file#starting-two-servers). I will look at seeing if start-server-and-test works here as well.